shared/common-adapters/tab-bar.native.tsx

```diff
@@ -149,27 +162,27 @@ const styleBadgeNav = {
 }
 
 const stylesContainer = {
```

Comment:
should these guys go in a stylesheet create?

Reply:
Oh good catch, thanks!